src/components/biggive-carousel/biggive-carousel.tsx

```diff
@@ -0,0 +1,103 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-carousel',
+  styleUrl: 'biggive-carousel.scss',
+  shadow: true,
+})
+export class BiggiveCarousel {
+  @Element() host: HTMLBiggiveCarouselElement;
+
+  @Prop() spaceBelow: spacingOption = 4;
+
+  @Prop() columnCount: number = 3;
+
+  @Prop() buttonBackgroundColour: brandColour = 'white';
+
+  @Prop() buttonIconColour: brandColour = 'primary';
+
+  currentTab = 0;
+  itemCount = 0;
+  itemWidth = 0;
+  max = 0;
+  min = 0;
+  sleeve;
```

Comment:
Could we add a type on `sleave`? I think the 5 things above it don't necessarily need a type on all them as `number` by widening from `0`.

Reply:
Updated as per your suggestion